server/serverMethods.js

```diff
@@ -418,6 +418,7 @@ function ServerMethods(aLogLevel, aModules) {
         ATPrimaryCategory: aReq.tbConfig.ATPrimaryCategory,
         ATSiteIdentifier: aReq.tbConfig.ATSiteIdentifier,
         ATFunctionDept: aReq.tbConfig.ATFunctionDept,
+        maxUsersPerRoom: aReq.tbConfig.maxUsersPerRoom,
```

Comment:
Does this change belong to this pr?

Reply:
yes. there is an existing test that was failing. fixing this revealed other broken tests